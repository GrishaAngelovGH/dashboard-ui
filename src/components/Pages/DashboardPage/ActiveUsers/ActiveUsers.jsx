import Statistic from 'antd/lib/statistic'
import Row from 'antd/lib/row'
import Col from 'antd/lib/col'
import { LikeOutlined } from '@ant-design/icons'

const ActiveUsers = () => (
	<div className='row g-0 justify-content-center text-center p-3'>
		<div className='col-md-9 ml-auto'>
			<Row gutter={5}>
				<Col span={4} xs={12} sm={9} lg={4}>
					<Statistic title='Active Users' value={112893} />
				</Col>
				<Col span={4} xs={12} sm={9} lg={4}>
					<Statistic title='Generated Revenue' value={112893} precision={2} />
				</Col>
				<Col span={4} xs={12} sm={9} lg={4}>
					<Statistic title='Feedback' value={1128} prefix={<LikeOutlined />} />
				</Col>
				<Col span={4} xs={12} sm={9} lg={4}>
					<Statistic title='Completed Tasks' value={93} suffix='/ 100' />
				</Col>
			</Row>
		</div>
	</div>
)

export default ActiveUsers