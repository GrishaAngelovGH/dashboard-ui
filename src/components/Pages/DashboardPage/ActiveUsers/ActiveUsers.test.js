import { render } from '@testing-library/react'

import ActiveUsers from './ActiveUsers'

window.matchMedia = window.matchMedia || function () {
    return {
        matches: false,
        addListener: function () { },
        removeListener: function () { }
    }
}

test('should render component', () => {
    const container = render(<ActiveUsers />)

    expect(container).toMatchSnapshot()
})