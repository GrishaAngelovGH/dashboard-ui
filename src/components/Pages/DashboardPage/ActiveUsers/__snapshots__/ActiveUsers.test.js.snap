// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ActiveUsers component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row no-gutters justify-content-center text-center p-3"
      >
        <div
          class="col-md-9 ml-auto"
        >
          <div
            class="ant-row"
            style="margin-left: -2.5px; margin-right: -2.5px;"
          >
            <div
              class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4"
              style="padding-left: 2.5px; padding-right: 2.5px;"
            >
              <div
                class="ant-statistic"
              >
                <div
                  class="ant-statistic-title"
                >
                  Active Users
                </div>
                <div
                  class="ant-statistic-content"
                >
                  <span
                    class="ant-statistic-content-value"
                  >
                    <span
                      class="ant-statistic-content-value-int"
                    >
                      112,893
                    </span>
                  </span>
                </div>
              </div>
            </div>
            <div
              class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4"
              style="padding-left: 2.5px; padding-right: 2.5px;"
            >
              <div
                class="ant-statistic"
              >
                <div
                  class="ant-statistic-title"
                >
                  Generated Revenue
                </div>
                <div
                  class="ant-statistic-content"
                >
                  <span
                    class="ant-statistic-content-value"
                  >
                    <span
                      class="ant-statistic-content-value-int"
                    >
                      112,893
                    </span>
                    <span
                      class="ant-statistic-content-value-decimal"
                    >
                      .00
                    </span>
                  </span>
                </div>
              </div>
            </div>
            <div
              class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4"
              style="padding-left: 2.5px; padding-right: 2.5px;"
            >
              <div
                class="ant-statistic"
              >
                <div
                  class="ant-statistic-title"
                >
                  Feedback
                </div>
                <div
                  class="ant-statistic-content"
                >
                  <span
                    class="ant-statistic-content-prefix"
                  >
                    <span
                      aria-label="like"
                      class="anticon anticon-like"
                      role="img"
                    >
                      <svg
                        aria-hidden="true"
                        data-icon="like"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="64 64 896 896"
                        width="1em"
                      >
                        <path
                          d="M885.9 533.7c16.8-22.2 26.1-49.4 26.1-77.7 0-44.9-25.1-87.4-65.5-111.1a67.67 67.67 0 00-34.3-9.3H572.4l6-122.9c1.4-29.7-9.1-57.9-29.5-79.4A106.62 106.62 0 00471 99.9c-52 0-98 35-111.8 85.1l-85.9 311H144c-17.7 0-32 14.3-32 32v364c0 17.7 14.3 32 32 32h601.3c9.2 0 18.2-1.8 26.5-5.4 47.6-20.3 78.3-66.8 78.3-118.4 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7-.2-12.6-2-25.1-5.6-37.1zM184 852V568h81v284h-81zm636.4-353l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 22.4-13.2 42.6-33.6 51.8H329V564.8l99.5-360.5a44.1 44.1 0 0142.2-32.3c7.6 0 15.1 2.2 21.1 6.7 9.9 7.4 15.2 18.6 14.6 30.5l-9.6 198.4h314.4C829 418.5 840 436.9 840 456c0 16.5-7.2 32.1-19.6 43z"
                        />
                      </svg>
                    </span>
                  </span>
                  <span
                    class="ant-statistic-content-value"
                  >
                    <span
                      class="ant-statistic-content-value-int"
                    >
                      1,128
                    </span>
                  </span>
                </div>
              </div>
            </div>
            <div
              class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4"
              style="padding-left: 2.5px; padding-right: 2.5px;"
            >
              <div
                class="ant-statistic"
              >
                <div
                  class="ant-statistic-title"
                >
                  Completed Tasks
                </div>
                <div
                  class="ant-statistic-content"
                >
                  <span
                    class="ant-statistic-content-value"
                  >
                    <span
                      class="ant-statistic-content-value-int"
                    >
                      93
                    </span>
                  </span>
                  <span
                    class="ant-statistic-content-suffix"
                  >
                    / 100
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row no-gutters justify-content-center text-center p-3"
    >
      <div
        class="col-md-9 ml-auto"
      >
        <div
          class="ant-row"
          style="margin-left: -2.5px; margin-right: -2.5px;"
        >
          <div
            class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4"
            style="padding-left: 2.5px; padding-right: 2.5px;"
          >
            <div
              class="ant-statistic"
            >
              <div
                class="ant-statistic-title"
              >
                Active Users
              </div>
              <div
                class="ant-statistic-content"
              >
                <span
                  class="ant-statistic-content-value"
                >
                  <span
                    class="ant-statistic-content-value-int"
                  >
                    112,893
                  </span>
                </span>
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4"
            style="padding-left: 2.5px; padding-right: 2.5px;"
          >
            <div
              class="ant-statistic"
            >
              <div
                class="ant-statistic-title"
              >
                Generated Revenue
              </div>
              <div
                class="ant-statistic-content"
              >
                <span
                  class="ant-statistic-content-value"
                >
                  <span
                    class="ant-statistic-content-value-int"
                  >
                    112,893
                  </span>
                  <span
                    class="ant-statistic-content-value-decimal"
                  >
                    .00
                  </span>
                </span>
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4"
            style="padding-left: 2.5px; padding-right: 2.5px;"
          >
            <div
              class="ant-statistic"
            >
              <div
                class="ant-statistic-title"
              >
                Feedback
              </div>
              <div
                class="ant-statistic-content"
              >
                <span
                  class="ant-statistic-content-prefix"
                >
                  <span
                    aria-label="like"
                    class="anticon anticon-like"
                    role="img"
                  >
                    <svg
                      aria-hidden="true"
                      data-icon="like"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="64 64 896 896"
                      width="1em"
                    >
                      <path
                        d="M885.9 533.7c16.8-22.2 26.1-49.4 26.1-77.7 0-44.9-25.1-87.4-65.5-111.1a67.67 67.67 0 00-34.3-9.3H572.4l6-122.9c1.4-29.7-9.1-57.9-29.5-79.4A106.62 106.62 0 00471 99.9c-52 0-98 35-111.8 85.1l-85.9 311H144c-17.7 0-32 14.3-32 32v364c0 17.7 14.3 32 32 32h601.3c9.2 0 18.2-1.8 26.5-5.4 47.6-20.3 78.3-66.8 78.3-118.4 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7-.2-12.6-2-25.1-5.6-37.1zM184 852V568h81v284h-81zm636.4-353l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 22.4-13.2 42.6-33.6 51.8H329V564.8l99.5-360.5a44.1 44.1 0 0142.2-32.3c7.6 0 15.1 2.2 21.1 6.7 9.9 7.4 15.2 18.6 14.6 30.5l-9.6 198.4h314.4C829 418.5 840 436.9 840 456c0 16.5-7.2 32.1-19.6 43z"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  class="ant-statistic-content-value"
                >
                  <span
                    class="ant-statistic-content-value-int"
                  >
                    1,128
                  </span>
                </span>
              </div>
            </div>
          </div>
          <div
            class="ant-col ant-col-4 ant-col-xs-12 ant-col-sm-9 ant-col-lg-4"
            style="padding-left: 2.5px; padding-right: 2.5px;"
          >
            <div
              class="ant-statistic"
            >
              <div
                class="ant-statistic-title"
              >
                Completed Tasks
              </div>
              <div
                class="ant-statistic-content"
              >
                <span
                  class="ant-statistic-content-value"
                >
                  <span
                    class="ant-statistic-content-value-int"
                  >
                    93
                  </span>
                </span>
                <span
                  class="ant-statistic-content-suffix"
                >
                  / 100
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
